nirc_ehr/resources/queries/ehr_lookups/cage.js

```diff
@@ -0,0 +1,6 @@
+var triggers = require("ehr/triggers");
+triggers.initScript(this);
+
+function onUpsert(helper, scriptErrors, row, oldRow) {
+    row.location = row.status;
```

Comment:
This is to work around the concatenation in the core EHR trigger script, correct? Can you add a comment?

Reply:
sure